frontend/src/components/University/MainUniversitySidebar.jsx

```
@@ -0,0 +1,47 @@
+  }, []);
+
+  return (
+    <div className="w-1/4 p-4 border-r overflow-y-auto h-screen">
+      {universities.map((uni) => (
+        <div key={uni.id} className="mb-4">
+          <h2
+            className="text-lg font-semibold cursor-pointer text-blue-600 hover:underline"
+            onClick={() => {
+                console.log("Navigating to:", `/university/${uni.name}`);
+                navigate(`/university/${uni.name}`)}}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)